### PR TITLE
[Foundation] Rearrange the top of the work packages form

### PR DESCRIPTION
**Pull request by floriank**
As requested, this rearranges the top part of the work packages form.

Meets a quick win requirement of https://community.openproject.org/work_packages/19073

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://floriank-fix-19073-rearrange-work-package-form.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://floriank-fix-19073-rearrange-work-package-form.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: spec/features/work_packages/new_work_package_spec.rb [hunk 1]
```diff
@@ -46,8 +46,8 @@
 
         work_packages_page.visit_new
 
-        input = page.find('#work_package_start_date')
-        input.click
+        # Fill in the date, a sa simple click does not seem to trigger the datepicker here
+        fill_in "Start date", with: DateTime.now.strftime('%Y-%m-%d')
```
Comment (myabc) [inline spec/features/work_packages/new_work_package_spec.rb]: @NobodysNightmare @houndci gets it right here.

**Comment by myabc** [comment]
@floriank I don't agree that this looks better, but it meets the specification, so will merge as soon as we get a green build.

**Comment by floriank** [comment]
Me neither - it was request by @lindenthal. 

It will be a temporary solution anyway.

**Pull request merged**